src/vm/ceemain.cpp

```diff
@@ -1462,6 +1463,12 @@ void STDMETHODCALLTYPE EEShutDownHelper(BOOL fIsDllUnloading)
     DiagnosticServer::Shutdown();
 #endif // FEATURE_PERFTRACING
 
+    // When running under FEATURE_PAL, the SetCommandLineArgs call above will
+    // call SaveManagedCommandLine which will allocate memory using new WCHAR[]
+    // We can release this memory now.
+    //
+    ReleaseManagedCommandLine();
```

Comment:
We are intentionally not doing any cleanup. We depend on the OS to take care of releasing the memory.

Reply:
I have now removed the new method: ReleaseManagedCommandLine();